examples/custom-animation/running-dashed-message-flow/README.md

```diff
@@ -32,9 +32,5 @@ It looks like `<path d="..." fill="none" stroke="black" stroke-width="1.78" stro
 ````
 
 ```javascript
-    const animatedMessageFlowElt = bpmnVisualization.bpmnElementsRegistry.getElementsByIds(['message_flow_id'])[0].htmlElement;
-    animatedMessageFlowElt.classList.add('running-dashed');
+    bpmnVisualization.bpmnElementsRegistry.addCssClasses(['message_4'], 'running-dashed');
```

Comment:
❓ I guess we have the same remark here about the element id

Reply:
ah yes I get it :)